cmd/cstor-pool-mgmt/pool/pool.go

```diff
@@ -50,6 +50,12 @@ var PoolAddEventHandled = false
 // PoolNamePrefix is a typed string to store pool name prefix
 type PoolNamePrefix string
 
+// CStorPools is a map of string, API version of CStorPool
+var CStorPools map[string]*apis.CStorPool
+
+// CStorZpools is a map of string, backend topology of CStorPool
+var CStorZpools map[string]Topology
```

Comment:
We might want to think of some name based on the functionality this variable provides.
Same as above comment.